ios-competency-test/ios-competency-test/FriendCommand.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  FriendCommand.swift
+//  ios-competency-test
+//
+//  Created by Sam Gubler on 3/27/15.
+//  Copyright (c) 2015 Sam Gubler. All rights reserved.
+//
+
+import UIKit
+import Alamofire
```

Comment:
I forgot to remove that line.  I got hung up on JSON parsing for a moment because I was trying to use json["status"]?.stringValue which is incorrect.  It should use as? String.  So I went down a path of thinking I was getting some strange stuff from Alamofire so I dropped it out.  I just forgot to clean up my reference (interesting that it compiles with the framework removed).  I had added it as an embedded framework though.